doc/grin4bitcoiners.md

```diff
@@ -8,7 +8,7 @@ There are 3 main properties of Grin transactions that make them private:
 2. There are no amounts.
 3. 2 transactions, one spending the other, can be merged in a block to form only one, removing all intermediary information.
 
-The 2 first properties mean that all transactions look the same. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).
+The 2 first properties mean that all transactions look similar. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).
```

Comment:
I think this should be `the same`. They are not just similar - they all look exactly like random data.

Reply:
That's why I changed to `similar`. They look similar but not `the same`. The same for me means exactly the same.

The Cambridge dictionary says [here](https://dictionary.cambridge.org/grammar/british-grammar/adjectives/same-similar-identical)
> Same means that two or more things are exactly like one another.

> We use similar if two or more things are not entirely the same, or identical if two or more things are exactly the same.

Reply:
Not _identical_ in that they are all the same exact piece of random data - but they're not similar to random data, they _are_ random data.

Reply:
Maybe we want to say "indistinguishable" here and remove the ambiguity.
But we're getting close to cryptography terminology there that I'll defer to other people for.

Reply:
ah! I was just writing that.
I can leave `the same` if you want :-)

Reply:
Yeah that's what I understood when I changed `the same` to `similar`.
How about (last try I promise):
`The 2 first properties mean that all transactions are indistinguishable from one another. Unless you directly participated in the transaction, all inputs and outputs look like random pieces of data (in lingo, they're all random curve points).`

Reply:
:+1: